packages/lisk-transactions/src/constants.js

```diff
@@ -22,6 +22,10 @@ export const SIGNATURE_FEE = 5 * FIXED_POINT;
 export const DELEGATE_FEE = 25 * FIXED_POINT;
 export const VOTE_FEE = 1 * FIXED_POINT;
 export const MULTISIGNATURE_FEE = 5 * FIXED_POINT;
+export const MULTISIGNATURE_MAX_LIFETIME = 72;
+export const MULTISIGNATURE_MIN_LIFETIME = 1;
+export const MULTISIGNATURE_MAX_SIGNATURES = 15;
```

Comment:
@shuse2 Please confirm this value.

Reply:
so it should be ok
https://github.com/LiskHQ/lisk/blob/development/config/default/constants.js#L80-L92